src/ui-lib/sass/05-components/01-atoms/04-media/_24-image.scss

```diff
@@ -0,0 +1,7 @@
+.grav-c-image {
+  &__invertible {
+    @media (prefers-color-scheme: dark) {
+      filter: invert(100);
```

Comment:
I think this is fine. Especially since it's opt-in via the modifier class.

Reply:
👍